tests/Unit/ParallelAlgorithms/Interpolation/Test_CleanUpInterpolator.cpp

```diff
@@ -37,13 +35,17 @@ namespace intrp {
 
 namespace {
 
+struct OtherId : db::SimpleTag {
```

Comment:
Why is this called `OtherId`? Why not `TimeId` or something that has `Time` in it?

Reply:
Looking at the code again, I believe that the only reason that there are 2 tags in this test is to test that interpolation works with both `TimeStepId` and `double`.  So if interpolation only works with `double`, then I think there doesn't need to be a 2nd tag at all.